src/app/core/services/label.service.ts

```diff
@@ -17,17 +17,43 @@ const DISPLAY_NAME_SEVERITY = 'Severity';
 const DISPLAY_NAME_BUG_TYPE = 'Bug Type';
 const DISPLAY_NAME_RESPONSE = 'Response';
 
+// The HTML template definition of selected labels are hard-coded here, move to a config file in the future
+const VERY_LOW_DEFINITION = '<p>A flaw that is <mark>purely cosmetic</mark> and <mark>does not affect usage</mark>. For example, '
+  + '<ul>'
+  + '<li>a typo issues</li>'
```

Comment:
~a~ ?

Reply:
Opps, I forgot to remove it out when I tried out the list style. Thank you for pointing that out!